designs/add-recap-search-option-about-representation/party types that will be considered to be petitioners.md

```diff
@@ -0,0 +1,356 @@
+[This proposed change](./readme.md) will include Django code that will examine each docket party's type to determine if the party is what we will consider to be a petitioner.
+
+In the list below, the party types that the Django code will consider to identify a petitioner are `highlighted like this` <<<<<.
```

Comment:
I haven't read these over. @johnhawkinson, have you? I'm sure you'll do a better job, but I can give it a go if you haven't already (or don't want to).